lib/api/handler/pokemon_api.dart

```diff
@@ -9,5 +13,19 @@ class PokemonApi {
   final ApiClient apiClient;
   final Uri baseUri;
 
-//TODO: to be continued...
-}
+  Future<List<Pokemon>> getPokemonList({
+    required String offset,
+    required String limit,
+  }) async {
+    final queryParameters = <String, dynamic>{};
+    queryParameters[queryParamLimit] = limit;
+    queryParameters[queryParamOffset] = offset;
+    final uri = baseUri.replace(
+        queryParameters: queryParameters, path: '${baseUri.path}$endPointPokemon');

```

Comment:
I've mentioned this already in your previous pr - again do you need the whitespace here? else remove it 🙂